components/form/LabeledInput.vue

```diff
@@ -36,7 +36,7 @@ export default {
       type:    Boolean,
     },
 
-    minHeight: {
+    minInputHeight: {
```

Comment:
I think this one can now stay as is, I changed the type to `Number` so it matches the underlying type

Reply:
It refers to a css property like "50px" so should be a string